cypress/integration/unit_tests/manage_disaster_set_score_parameters_test.js

```diff
@@ -62,6 +61,20 @@ describe('Score parameters-related tests for manage_disaster.js', () => {
     cy.get('#damage-asset-select').should('have.value', '');
     cy.get('#map-bounds-div').should('be.visible');
     cy.get('.score-bounds-delete-button').should('not.be.visible');
+    // Has NY in view after EE promise finishes and zoom happens.
+    cy.wrap(scoreBoundsMap.stateBoundsPromise).then(() => {
+      new Promise(
+          (resolve) => google.maps.event.addListenerOnce(
+              scoreBoundsMap.map, 'zoom_changed', () => {
+                expect(scoreBoundsMap.map.getBounds().contains({
+                  lng: -74,
```

Comment:
Maybe add that these came from inspecting NY bounds (assuming that's how it happened?)

Reply:
Comment added.